aptos-move/framework/aptos-stdlib/sources/hash.move

```diff
@@ -1,4 +1,11 @@
-/// Non-cryptographic hashes
+/// Cryptographic hashes:
+/// - Keccak-256: see https://keccak.team/keccak.html
+///
+/// In addition, SHA2-256 and SHA3-256 are available in `std::hash`. Note that SHA3-256 is a variant of Keccak: it is
```

Comment:
How is sha3-256 generated using std::hash?  only see sip_hash and keccak256 functions in this file.